server/item/bottle_of_enchanting.go

```diff
@@ -11,7 +11,9 @@ type BottleOfEnchanting struct{}
 // Use ...
 func (b BottleOfEnchanting) Use(tx *world.Tx, user User, ctx *UseContext) bool {
 	create := tx.World().EntityRegistry().Config().BottleOfEnchanting
-	opts := world.EntitySpawnOpts{Position: eyePosition(user), Velocity: user.Rotation().Vec3().Mul(0.7)}
+	r := user.Rotation()
+	r[1] = throwableOffset(r[1])
+	opts := world.EntitySpawnOpts{Position: eyePosition(user), Velocity: r.Vec3().Mul(0.6)}
```

Comment:
This is not a mistake by the way. For some reason, bottle o' enchanting needed a higher throwing force (`0.6`) than splash and lingering potions (`0.5`) to match vanilla.